hazelcast/token_provider.py

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
Would it make sense to rename this module to security, so that we can put things related to security here in the future, if we want to improve the functionality we have?

Reply:
OK: https://github.com/hazelcast/hazelcast-python-client/pull/446/commits/36946e76b40c9e937d31840499db6bd23f1b14c6